rotkehlchen/chain/ethereum/decoding/decoder.py

```diff
@@ -461,7 +461,7 @@ def _maybe_decode_simple_transactions(
         if amount == ZERO:
             return events
 
-        events.append(HistoryBaseEntry(
+        eth_sent_event = HistoryBaseEntry(
```

Comment:
why this change?

Reply:
Because I reverted the changes I made here with the eth transfers but forgot to undo this one. Should be fixed now

Reply:
I see. Expected something like that but was not sure.